### PR TITLE
Move yaml tempalte to models dir

### PR DESCRIPTION
**Pull request by yaacov**
Move kubevirt-plugin models into a separate dir.

Ref:
https://github.com/openshift/console/pull/1673#discussion_r297594646

### CODE REVIEW

**Comment by yaacov** [comment]
This was requested by @vojtechszocs @suomiy .

@mareklibra @vojtechszocs @suomiy @rawagner please review.

File: frontend/packages/kubevirt-plugin/src/models/yaml-templates.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Map as ImmutableMap } from 'immutable';
 
 import { TemplateModel } from '@console/internal/models';
-import { VirtualMachineModel } from './models';
+import { VirtualMachineModel } from '.';
 
 export const yamlTemplates = ImmutableMap()
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/models/yaml-templates.ts]: can we go one level deeper to `models/templates/` and have a separate file for each template?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/models/yaml-templates.ts]: make sense to me too ... :+1:

File: frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml-templates.ts [hunk 2]
```diff
@@ -1,50 +1,10 @@
...
-            image: kubevirt/cirros-registry-disk-demo
-        - name: cloudinitdisk
-          cloudInitNoCloud:
-            userDataBase64: SGkuXG4=
-`,
-  )
-  .setIn(
-    [TemplateModel, 'vm-template'],
-    `
+export const VmTemplateYAMLTemplates = ImmutableMap().setIn(
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml-templates.ts]: please rename to either `VirtualMachineTemplate...` or `VMTemplate...`

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml-templates.ts]: :+1: Nice

File: frontend/packages/kubevirt-plugin/src/models/templates/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export * from './vm-template-yaml-templates';
+export * from './vm-yaml-templates';
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/models/templates/index.ts]: nit: as this is already under `models/templates`, do we need the extra `-templates` at the end of the filename?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/models/templates/index.ts]: :+1:

**Pull request merged**